### PR TITLE
Update HLAPI docs.

### PR DESCRIPTION
**Pull request by shadow001996**
Updating docs after adding set selection API to HLAPIs.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/api/classes/Editor.md [hunk 1]
```diff
@@ -157,11 +157,10 @@ ___
 ▸ **createStroke**(`options?`): [`Stroke`](../interfaces/Stroke.md)
 
 See [StrokeOptions](../interfaces/StrokeOptions.md) for more details on the `options` fields. Defaults:
-
 - `color` has default value DEFAULT_STROKE_COLOR if none is provided.
 - `width` has default value DEFAULT_STROKE_WIDTH if none is provided.
-- `dashPattern` has default value [] if none is provided. Array must be
-  of even length. Values cannot be negative.
+- `dashPattern` has default value [] if none is provided. DashPattern array must be
+  of even length, otherwise it will throw error. Values cannot be negative.
```
Comment (peterflynn) [inline src/pages/api/classes/Editor.md:163]: This looks like an older copy of the docs.

File: src/pages/api/classes/Editor.md [hunk 2]
```diff
@@ -115,8 +115,8 @@ This local client will act as having unsaved changes until the upload has finish
 | Name | Type | Description |
 | :------ | :------ | :------ |
 | `bitmapData` | [`BitmapImage`](../interfaces/BitmapImage.md) | BitmapImage resource (e.g. returned from loadBitmapImage()). |
-| `options` | `Object` | Additional configuration: |
-| `options.initialSize?` | [`RectangleGeometry`](../interfaces/RectangleGeometry.md) | Size the image is displayed at. Must have the same aspect ratio as bitmapData. Defaults to the        size the image would be created at by a UI drag-drop gesture (typically the image's full size, but scaled down        if needed to stay below an application-defined size cap). |
+| `options` | `Object` | Additional configuration:      - initialSize - Size the image is displayed at. Must have the same aspect ratio as bitmapData. Defaults to the        size the image would be created at by a UI drag-drop gesture (typically the image's full size, but scaled down        if needed to stay below an application-defined size cap). |
+| `options.initialSize?` | [`RectangleGeometry`](../interfaces/RectangleGeometry.md) | - |
```
Comment (peterflynn) [inline src/pages/api/classes/Editor.md:119]: This is a formatting error coming from a bug in the docs generator. Please see the instructions in the README about regenerating docs where this is noted.

File: src/pages/api/enums/SceneNodeTypeValueID.md [hunk 3]
```diff
@@ -41,12 +43,28 @@ ___
...
 
 • **ellipse** = ``"Ellipse"``
 
 ___
 
+### <a id="gridLayout" name="gridLayout"></a> gridLayout
+
+• **gridLayout** = ``"GridLayout"``
+
+Type of GridLayoutNode represents a grid layout in the scenegraph used to create a layout grid that other content can be placed into
```
Comment (peterflynn) [inline src/pages/api/enums/SceneNodeTypeValueID.md:64]: I think this is just supposed to say "Type of GridLayoutNode" and nothing else – it looks like the sandbox copy of this got its JSDocs out of sync with the upstream original one somehow.

Could you do a quick double-check of everything else in SceneNodeTypeValueID too to make sure none of the other docs are stale also?

**Comment by nimitz92** [comment]
Aren't the docs moved to the other repo now?
They live here now:
https://github.com/AdobeDocs/express-add-ons-docs/tree/main/src/pages/references/authoring/editor

**Comment by peterflynn** [comment]
@nimitz92 yep, it looks like this is superseded by Priyanshu's newer PR into the newer repo: https://github.com/AdobeDocs/express-add-ons-docs/pull/26

Closing this one to avoid confusion.  @hollyschinsky should we just lock or archive this entire repo now?

**Pull request closed without merging**